cmd/internal/exec.go

```diff
@@ -23,8 +23,9 @@ import (
 	"strings"
 )
 
+// RunCmd prints the provided message and command and then executes it binding stdout and stderr
 func RunCmd(msg, cmd string, args ...string) error {
-	c := exec.Command(cmd, args...) // #nolint:gosec
+	c := exec.Command(cmd, args...) //nolint:gosec
```

Comment:
Why here was not added the space? (// nolint:gosec)

Reply:
See https://github.com/kubernetes-sigs/kubebuilder/pull/1368#discussion_r379111906.